pipelinewise/fastsync/partialsync/mysql_to_snowflake.py

```diff
@@ -113,7 +129,36 @@ def main_impl():
         ''', args.table, args.column, args.start_value, args.end_value
     )
 
-    sync_excs = partial_sync_table(args=args)
+    table_names = args.table.split(',')
+    column_names = args.column.split(',')
+
+    start_values = args.start_value.split(',')
+    if args.end_value:
+        end_values = args.end_value.split(',')
+    else:
+        end_values = [None] * len(table_names)
+    if args.drop_target_table:
+        drop_target_tables = [literal_eval(x) for x in args.drop_target_table.split(',')]
+    else:
+        drop_target_tables = [False] * len(table_names)
+
+    sync_tables = {}
+    for ind, table in enumerate(table_names):
+        sync_tables[table] = {
+            'column': column_names[ind],
+            'start_value': start_values[ind],
+            'end_value': end_values[ind],
+            'drop_target_table': drop_target_tables[ind],
+        }
```

Comment:
I see this repeated in another file, would it be possible to bundle this into a function?

Reply:
- [ ] Introduce a method for it